asm/trafficdirector/td-galley.yaml

```diff
@@ -61,7 +61,7 @@ spec:
         - --validation-port=9443
         - --sinkAuthMode=$(SINKAUTHMODE)
         - --sinkAddress=$(SINKADDRESS)
-        - --sinkMeta=project_id=$(PROJECT_ID),sds_path=$(SDSPATH),trace=0,enable_security_beta=false # {"$ref":"#/definitions/io.k8s.cli.substitutions.sink-meta"}
+        - --sinkMeta=project=$(PROJECT_ID),location=$(GCP_LOCATION),network=projects/$(PROJECT_ID)/global/networks/default,sds_path=$(SDSPATH),trace=0,enable_security_beta=true # {"$ref":"#/definitions/io.k8s.cli.substitutions.sink-meta"}
```

Comment:
enable_security_beta=false

Reply:
Better remove it completely, Galley is not supposed to set this at all.

Reply:
It turns out we use it for testing, so please leave it but change it to false.

Reply:
Set to false.